{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40792198",
  "multipleFaultNumbers": null,
  "date": "2020-12-29T14:42:39",
  "region": "Bolton",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-29T14:42:39",
  "information": "",
  "actualTimeOfRestoration": "2020-12-29T14:42:39",
  "outageCentrePoint": {
    "lat": 53.54582,
    "lng": -2.434584
  },
  "outageLocations": null,
  "AffectedPostcodes": " BL4 0PF, BL4 0PG, BL4 0PP, BL4 0PQ, BL4 0PR, BL4 0PS, BL4 0PT, BL4 0PU, BL4 0PX, BL4 0PY, BL4 0PZ, BL4 0QA, BL4 0QW, BL4 0RJ, BL4 0RL",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GS"
}